lib/jsdom/level2/html.js

```diff
@@ -164,11 +166,46 @@ ResourceQueue.prototype = {
   }
 };
 
+function RequestManager(options) {
+  this.pool = options.pool;
```

Comment:
This is used by XMLHttpRequest and resource-loader, I could put them in the document directly with symbols but putting them inside the request manager seems more logical to me.